modules/apps/29-fee/keeper/events.go

```diff
@@ -0,0 +1,24 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/ibc-go/v3/modules/apps/29-fee/types"
+	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+)
+
+// Emit event so that relayers know an incentivized packet is ready to be relayed
+func EmitIncentivizedPacket(ctx sdk.Context, identifiedFee types.IdentifiedPacketFee) {
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(
+			types.EventTypeIncentivizedPacket,
+			sdk.NewAttribute(channeltypes.AttributeKeyPortID, identifiedFee.PacketId.PortId),
+			sdk.NewAttribute(channeltypes.AttributeKeyChannelID, identifiedFee.PacketId.ChannelId),
+			sdk.NewAttribute(channeltypes.AttributeKeySequence, fmt.Sprint(identifiedFee.PacketId.Sequence)),
+			sdk.NewAttribute(types.AttributeKeyRecvFee, fmt.Sprint(identifiedFee.Fee.RecvFee)),
+			sdk.NewAttribute(types.AttributeKeyAckFee, fmt.Sprint(identifiedFee.Fee.AckFee)),
+			sdk.NewAttribute(types.AttributeKeyTimeoutFee, fmt.Sprint(identifiedFee.Fee.TimeoutFee)),
```

Comment:
Question: Does it make sense to emit the cumulative fee for each type here?

It would be useful for now, but we may have to revert back if relayer list becomes used.

cc: @colin-axner

Reply:
It might be hard to do with the way we plan to iterate over the identified fees. I don't think validators should be running extra logic so events are presented. Is there a clean way of aggregating the fees?

Reply:
Not really, I think it requires some iteration. Though only one retrieval from store

Reply:
But in the case that the relayers are different?

Reply:
If we do this I think we should do this in a separate PR for adding some helper functions for aggregating fees & it also gives us time to think if we want it.